plugins/com.google.cloud.tools.eclipse.appengine.login/src/com/google/cloud/tools/eclipse/appengine/login/GoogleLoginService.java

```diff
@@ -28,17 +42,59 @@
  */
 public class GoogleLoginService {
 
-  // TODO(chanseok): this will be needed later.
-  public static final List<String> OAUTH_SCOPES = Collections.unmodifiableList(Arrays.asList(
-      "https://www.googleapis.com/auth/userinfo#email"
+  private static final List<String> OAUTH_SCOPES = Collections.unmodifiableList(Arrays.asList(
```

Comment:
Perhaps this functionality should be in `com.google.cloud.tools.eclipse.`_appengine_`.login` instead of `com.google.cloud.tools.eclipse.login` — and if so, this list shouldn't be hardcoded here.  Or maybe renamed like `APPENGINE_SCOPES`?